### PR TITLE
Customer logging

### PR DESCRIPTION
**Pull request by Anntroy**

### CODE REVIEW

File: business_register/converter/kved.py [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
+logger.setLevel(logging.DEBUG)
+
+
 class KvedConverter(Converter):
     LOCAL_FILE_NAME = "kved.json"
 
     def __init__(self):
-        API_ADDRESS_FOR_DATASET = Register.objects.get(source_register_id=
-        "e1afb81c-70e4-4009-96a0-b240c36e4603").api_address
+        API_ADDRESS_FOR_DATASET = Register.objects.get(
```
Comment (roman-tiukh) [inline business_register/converter/kved.py:18]: you forgot `self.`
```suggestion
        self.API_ADDRESS_FOR_DATASET = Register.objects.get(
```

File: location_register/converter/koatuu.py [hunk 2]
```diff
@@ -95,17 +102,21 @@ def get_category_id(self, string):
...
+        except TypeError:
+            logger.exception('Tried to iterate NonType object')
+        else:
+            logger.info("Koatuu values saved")
 
     # writing entry to koatuu field in region table
     def save_to_region_table(self, data):
         region_dict = self.create_dictionary_for(Region)
-        for index, object_koatuu in enumerate(data):
+        for _, object_koatuu in enumerate(data):
```
Comment (roman-tiukh) [inline location_register/converter/koatuu.py:119]: if you dont need index, remove enumerate...

File: location_register/converter/koatuu.py [hunk 3]
```diff
@@ -114,14 +125,14 @@ def save_to_region_table(self, data):
...
-        print("Koatuu values to region saved")
+        logger.info("Koatuu values to region saved")
 
     # writing entry to koatuu field in district table and level_two records of city_table
     def save_to_district_table(self, data):
         region_dict = self.create_dictionary_for(Region)
         district_dict = self.create_dictionary_for_district_table(District)
         city_dict = self.create_dictionary_for_city_table(City)
-        for index, object_koatuu in enumerate(data):
+        for _, object_koatuu in enumerate(data):
```
Comment (roman-tiukh) [inline location_register/converter/koatuu.py:135]: same as prev comment

File: location_register/converter/koatuu.py [hunk 4]
```diff
@@ -154,7 +165,7 @@ def save_to_city_or_citydistrict(self, data):
         city_dict = self.create_dictionary_for_city_table(City)
         citydistrict_dict = self.create_dictionary_for_citydistrict_table()
         # getting values in json file Koatuu
-        for index, object_koatuu in enumerate(data):
+        for _, object_koatuu in enumerate(data):
```
Comment (roman-tiukh) [inline location_register/converter/koatuu.py:168]: same as prev comment

File: location_register/converter/koatuu.py [hunk 5]
```diff
@@ -241,7 +252,7 @@ def save_to_city_or_citydistrict_table(
     # writing entry to category fields where koatuu is empty
     def writing_category_null_id(self, table_model):
         category = self.get_category_id('null')
-        table_model_objects = table_model.objects.filter(category_id__isnull=True)
+        table_model_objects = table_model.objects.filter(category_id__isnull=True).count()
         for table_record in table_model_objects:
             table_record.category_id = category
             table_record.save()
```
Comment (roman-tiukh) [inline location_register/converter/koatuu.py:258]: this is not working code

File: business_register/converter/kved.py (hunk 1 above)
Comment (roman-tiukh) [inline business_register/converter/kved.py:18]:
```suggestion
       self.API_ADDRESS_FOR_DATASET = Register.objects.get(
```

File: data_converter/settings_local.base.py [hunk 6]
```diff
@@ -117,3 +117,81 @@
 #         'schedule': crontab(hour=1, minute=10, day_of_week=6),
 #     }
 # }
+
+# sentry configarution
+import sentry_sdk
+from sentry_sdk.integrations.django import DjangoIntegration
+from sentry_sdk.integrations.celery import CeleryIntegration
+from sentry_sdk.integrations.redis import RedisIntegration
```
Comment (roman-tiukh) [inline data_converter/settings_local.base.py:125]: move imports to top of file

File: data_converter/settings_local.base.py [hunk 7]
```diff
@@ -117,3 +117,81 @@
...
+    # django.contrib.auth) you may enable sending PII data.
+    send_default_pii=True
+)
+
+RAVEN_CONFIG = {
+    # format 'dsn':'https://<public_key>@sentry.io/<project_id>
+    "dsn":"<account_dsn>" # create your sentry account and add your own dsn
+}
+
+# to enable custommer logging
```
Comment (roman-tiukh) [inline data_converter/settings_local.base.py:143]: what is that, custommer? maybe you mean custom?

File: location_register/converter/koatuu.py (hunk 5 above)
Comment (roman-tiukh) [reply location_register/converter/koatuu.py:258]: Do you test your code before push to repository?

**Comment by roman-tiukh** [comment]
Any requirements??

File: location_register/converter/koatuu.py (hunk 5 above)
Comment (Anntroy) [reply location_register/converter/koatuu.py:258]: I've added errors to check logging. I didn't fill register ratu, so it throws me an error before. You are right it mustn't be in here.

File: data_converter/settings_local.base.py (hunk 6 above)
Comment (Anntroy) [reply data_converter/settings_local.base.py]: Done

File: data_converter/settings_local.base.py (hunk 7 above)
Comment (Anntroy) [reply data_converter/settings_local.base.py]: Fixed

File: location_register/converter/koatuu.py (hunk 5 above)
Comment (roman-tiukh) [reply location_register/converter/koatuu.py]: ok, but you add `.count()` here..........

**Comment by Anntroy** [comment]
> I think we have to log all cases with invalid company, fop, kved, etc records

I've put IntegrityError just as an example. We can add more log cases.

Everyone works more closely with some converters, and can better know which errors must be logged. Also, we can't add too much logging, because it will slow down our app

**Pull request merged**